Bolts/Common/BFTask.h

```diff
@@ -29,6 +29,9 @@ extern NSInteger const kBFMultipleErrorsError;
  */
 extern NSString *const BFTaskMultipleExceptionsException;
 
+extern NSString *const BFTaskMultipleErrorsUserInfoKey; // NSArray<NSError *>
```

Comment:
Can we document this the same way as other constants documented?
This will translate in a better auto-generated documentation.